rllib/utils/torch_utils.py

```diff
@@ -374,12 +376,14 @@ def one_hot(x: TensorType, space: gym.Space) -> TensorType:
         ValueError: If the given space is not a discrete one.
 
     Examples:
+        >>> import torch
+        >>> import gym
+        >>> from ray.rllib.utils.torch_utils import one_hot
```

Comment:
thanks for the attention to details.
not the first time we provide examples without imports :)

Reply:
@gjoliver yeah, it's a super painful death march, but I'm now halfway through. will activate `make doctest` in CI once I'm done, so that we don't introduce new ones.

Reply:
and as a user I can attest that it's very inconvenient to look up imports all the time.

Reply:
yeah, I remember it was the case when we were dogfooding Ray Train. and I got super annoyed.